test/unit/extension_conversion.js

```diff
@@ -153,9 +163,15 @@ test('registerExtensionPrimitives', t => {
     runtime.on(Runtime.EXTENSION_ADDED, blocksInfo => {
         t.equal(blocksInfo.length, testExtensionInfo.blocks.length);
 
+        blocksInfo.forEach(blockInfo => {
+            // `true` here means "either an object or a non-empty string but definitely not null or undefined"
+            t.true(blockInfo.info, 'Every block and pseudo-block must have a non-empty "info" field');
```

Comment:
I think in other VM unit tests we use `t.ok` for the truthy test. I believe that's just a synonym for t.true... ¯\_(ツ)_/¯

Reply:
Yeah -- I felt like `t.true` was semantically more accurate in this case since usually we do `t.ok(something == desired)`, and in this case I really just want to check that it's "truthy". Really I kind of want `t.typeIsOneOf(blockInfo.info, ['object','string']);` but that doesn't exist.

I guess it could be `t.ok(-1 != ['object', 'string'].indexOf(typeof blockInfo.info))` but that seems even harder to read than the other options :(

Also, obligatory "you dropped this: \\"

Reply:
`t.ok(['object', 'string'].includes(typeof blockInfo.info))` perhaps?

Reply:
> Really I kind of want `t.typeIsOneOf(blockInfo.info, ['object','string']);` but that doesn't exist.

Agreed.

> I guess it could be `t.ok(-1 != ['object', 'string'].indexOf(typeof blockInfo.info))` but that seems even harder to read than the other options :(

Also agreed.

> Also, obligatory "you dropped this: \"

I think github ate it!